### PR TITLE
IDEX-3240 Maven Mojo allowing to create Che plugins

### PR DESCRIPTION
**Pull request by benoitf**

### CODE REVIEW

File: che-maven-plugin/src/test/projects/single/src/main/che/machines/machine1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2012-2015 Codenvy, S.A.
```
Comment (skabashnyuk) [inline che-maven-plugin/src/test/projects/single/src/main/che/machines/machine1/Dockerfile]: Why this file contains Codenvy copyright?

**Comment by benoitf** [comment]
Use of versions from che-depmgt

File: che-maven-plugin/pom.xml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>che-core-parent</artifactId>
+        <groupId>org.eclipse.che.core</groupId>
+        <version>3.13.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <artifactId>che-maven-plugin</artifactId>
+    <packaging>maven-plugin</packaging>
```
Comment (skabashnyuk) [inline che-maven-plugin/pom.xml]: Not sure about artifact name and location. @gazarenkov wdyt?

**Comment by skabashnyuk** [comment]
LGTM you can merge it in master

**Comment by benoitf** [comment]
closing PR as it needs to be added in che, not che-core

**Pull request closed without merging**